main.js

```diff
@@ -66,17 +87,55 @@ function setAPPListeners () {
  * Opens new window with index.html(Jitsi Meet is loaded in iframe there).
  */
 function createJitsiMeetWindow () {
-    jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
-    jitsiMeetWindow.loadURL(indexURL);
+  jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
+  jitsiMeetWindow.loadURL(indexURL);
+  microWindow = new BrowserWindow(microWindowOptions);
```

Comment:
Currently my plan is to load the micro window when the main window loads, but hidden at first. When the user minimizes /lose focus the main window, I was going to make it visible. For the performance sake, I might gonna make an option to disable micro mode, which prevents the micro window from loading.